workflow-testing/src/main/java/com/squareup/workflow1/testing/RealRenderTester.kt

```diff
@@ -117,7 +118,9 @@ internal class RealRenderTester<PropsT, StateT, OutputT, RenderingT>(
   }
 
   @OptIn(ExperimentalStdlibApi::class)
-  override fun render(block: (RenderingT) -> Unit): RenderTestResult<PropsT, StateT, OutputT> {
+  override fun render(
+    block: (RenderingT) -> Unit
+  ): RenderTestResult<PropsT, StateT, OutputT, RenderingT> = apply {
```

Comment:
Does this actually change anything?

Reply:
I need to plumb `RenderingT` through generics.

Reply:
Wow, I spaced the param type. But I meant using `apply {` rather than just returning `this`.

Reply:
Oh, probably it was because I meant it as a draft and left a room for some potential changes.

Reply:
I still don't like `apply` here.

- [ ] Nix on apply